CLAUDE.md

```
@@ -0,0 +1,192 @@
+cd api && npm run db:init     # Initialize D1 database
+cd api && npm run db:migrate  # Run migrations
+
+# Extension development
+# 1. Edit files in extension/
+# 2. chrome://extensions → Reload CaptureAI
+# 3. Test on a webpage
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)